playwright-e2e/package.json

```diff
@@ -16,15 +16,17 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
-    "@playwright/test": "^1.31.1",
+    "@playwright/test": "^1.33.0",
```

Comment:
Don't upgrade Playwright version in this PR because you would also need to update CI Playwright docker image version, update test screenshots on Dev, Test and docker container.